libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationActivity.java

```diff
@@ -118,6 +118,7 @@ private void extractConfiguration(NavigationViewOptions.Builder options) {
     MapboxNavigationOptions navigationOptions = MapboxNavigationOptions.builder()
       .unitType(preferences.getInt(NavigationConstants.NAVIGATION_VIEW_UNIT_TYPE,
         NavigationUnitType.TYPE_IMPERIAL))
+      .locale(getResources().getConfiguration().locale)
```

Comment:
Great call here using the configuration locale here instead of `Locale.getDefault()`